WORKSPACE

```diff
@@ -1,25 +1,18 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "ade51a315fa17347e5c31201fdc55aa5ffb913377aa315dceb56ee9725e620ee",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.16.6/rules_go-0.16.6.tar.gz",
+    urls = ["https://github.com/bazelbuild/rules_go/releases/download/0.18.5/rules_go-0.18.5.tar.gz"],
+    sha256 = "a82a352bffae6bee4e95f68a8d80a70e87f42c4741e6a448bec11998fcc82329",
```

Comment:
If you want to be able to support crossbuilds for darwin from a linux host using bazel 0.26+, you'll need to use a git checkout of bazelbuild/rules_go currently: https://github.com/kubernetes-sigs/cluster-api-provider-aws/commit/8e12155d0eb12deddb5e1b45558d92637c942dca